keyboards/walletburner/neuron/config.h

```diff
@@ -3,8 +3,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x62AC
+#define VENDOR_ID       0x5742 // WB
+#define PRODUCT_ID      0x6e75 // NU
```

Comment:
I'm almost definitely nitpicking here, but:

```suggestion
#define PRODUCT_ID      0x6E75 // NU
```